src/parser/cleanHtml.ts

```diff
@@ -0,0 +1,20 @@
+import { pasteHandler, serialize } from '@wordpress/blocks';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+export function cleanHtml( html: string ): string {
+	init();
+	const blocks = pasteHandler( {
+		mode: 'BLOCKS',
+		HTML: html,
+	} );
+	return serialize( blocks );
+}
+
+let isInitialized = false;
+function init() {
+	if ( isInitialized ) {
+		return;
+	}
+	registerCoreBlocks();
```

Comment:
🎉

Reply:
Yep! Thanks for the pointer, that was indeed the solution, without it the call to `pasteHandler()` fails.